doc/usecases/polyculture/search_plants.md

```diff
@@ -0,0 +1,36 @@
+# Use Case: Search Plants
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can search for plants to add to her map and sort results by different criterion.
```

Comment:
Maybe mention the search functionality's context within the app, such as "while editing their map" e.g. 
The user can search for plants to add to his map and sort results by different criteria **while editing his green space**

Reply:
Shouldn't this be clear from the noted scope?

Reply:
And such things definitely shouldn't be added in "Brief" :wink: